schemas/IDLs/org.gel.models.participant.avro/1.2.0/Ngis.avdl

```diff
@@ -44,9 +46,13 @@ protocol Ngis {
          */
         string clinicalIndicationTestTypeCode;
         /**
-        Clinical indication Test Name
+        Test Type Id
+        */
+        string testTypeId;
```

Comment:
can we use `testId` instead of `testTypeId`?

Reply:
testType should be used so it is consistent with the Elucidata persistent data model from test_type.test_type_uid corresponding to the ci_test_type_uid. See https://cnfl.extge.co.uk/display/NGIS/Test+Directory+Data+Model?preview=/83920765/99386079/Test%20Directory%20V10n.pdf

However, this particular field is being debated on its use. I am awaiting Richard Scott's response.